docker-compose-dev.yml

```diff
@@ -16,13 +16,12 @@ services:
       ENABLE_RELOAD: "1"
```

Comment:
I'm just putting this here at the top but can we please get a release note snippet?